hardware/Wireless-Products-Protocols-and-Applications.md

```diff
@@ -1,54 +1,61 @@
 - Chip name conventions:
 
   For Zigbee/Thread products, there are also many MG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
-![](files/HW-Wireless-Products-Protocols-and-Applications/NameConvention-2.png)
+  <div align="center">
+  <img src="files\HW-Wireless-Products-Protocols-and-Applications\NameConvention-2.png">  
+  </div>
+
   As you can see from the name convention, the Zigbee/Thread chips include 2 series such as MG12 and MG13. The Zigbee/Thread chips only have  P performance grade. The chips are Tx/Rx type which include sub-G band, 2.4GHz band, and sub-G& 2.4GHz bands. The chips are classified with 2 temperature range as -40°-+85° and -40°-+125°. And there are many packages(QFN32, QFN48, QFN68, and BGA125) to select. 
 - Applications:
 
   Home and Building Automation, Beaconing, Lighting and Switching, Asset Tracking, Remote Control via Internet, etc.
 
-## BLE products and applications
+# 3. Bluetooth products and applications
 - Introduction: 
 
   Add Bluetooth® 5, Bluetooth 5.1 or Bluetooth mesh connectivity into sports and fitness, consumer electronics, beacons, and smart home applications with Bluetooth SoCs, certified modules and software from Silicon Labs. Take advantage of the latest Bluetooth low energy technologies including Bluetooth 5 and Bluetooth mesh with Silicon Labs wireless stacks and advanced development tools, and explore pre-programmed module solutions with Bluetooth Xpress.
 - Chip name conventions:
 
   For BLE products, there are many BG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
- ![](files/HW-Wireless-Products-Protocols-and-Applications/NameConvention-3.png)
-
+  <div align="center">
+  <img src="files\HW-Wireless-Products-Protocols-and-Applications\NameConvention-3.png">  
+  </div>
+ 
   As you can see from the name convention, the BLE chips include 3 series such as FG1,FG12, and FG13. The BLE chips are classified with  P (Performance), B (Basic), and V (Value) performance grades. The BLE chips are Tx/Rx type which include sub-G band, 2.4GHz band, and sub-G& 2.4GHz bands. The BLE chips are classified with 2 temperature range as -40°-+85° and -40°-+125°. And there are many packages(QFN32, QFN48, QFN68, and BGA125) to select. 
 - Applications:
 
   Home Automation, Beaconing, Sport and Fitness, Asset Tracking, AoA/AoD, etc.
 
-## Z-Wave products and applications
+# 4. Z-Wave products and applications
 - Introduction: 
 
```

Comment:
Why aren't we also including ZGM130S module here?

Reply:
Added.